### ISSUE TITLE
Move package.json to inside webpack directory

### ISSUE THREAD

**Issue by justin808** [justin808/react-webpack-rails-tutorial#10]
We can have a top level package.json that loads the /webpack/package.json.

This keeps the top level tidier.

From Heroku support:

I agree - as long as an extra root-level package.json is okay with you, it's probably a better bet to just use the officially supported buildpack. You can trigger sub-installs like this (in package.json):

```
  "scripts": {
    "postinstall": "cd ./client && npm install && cd ../server && npm install"
   }
```

Also, I'd love to get your feedback on how that works with the upcoming version of the Node buildpack (codenamed "yoga"):
https://github.com/heroku/heroku-buildpack-nodejs/tree/yoga
If you decide to try it out, please let me know what you think!
Thanks,
Hunter

**Comment by bdefore** [comment]
I decided to try this out and it works without many changes. My Procfile.dev became:
```
webpack: cd ./client && webpack -w --config ./client/webpack.rails.config.js
```
(not that i renamed 'webpack' dir to 'client')

Overall I'm much happier with the separation, with dependencies defined and installed relative to client. It also has the benefit of making it more straightforward for porting this process over to an existing rails setup, since you could just copy over the 'client' directory, make a few Procfile customizations and you're good to go.

**Comment by justin808** [comment]
@bdefore Awesome. One advantage of the name client is that if you use zsh and type webpack to switch directories, well, webpack is a exe. If you can submit a PR, I'll get that integrated. Otherwise, @mbreining  or I will try to change it shortly.

**Comment by justin808** [comment]
I'm creating this change. @bdefore I'll notify you so that you might be able to comment on the PR.

**Comment by justin808** [comment]
@bdefore If you get a chance, please review Pull request #19.

**Comment by bdefore** [comment]
@justin808 i've left a couple notes but overall looks like what i'd expect. you'll also want to update your blog post references to 'webpack' directory since that may be how people come to find your tutorial.

**Comment by justin808** [comment]
Closed by #19.
